src/common/extensionMessaging.ts

```diff
@@ -35,7 +47,8 @@ export class ExtensionMessageSender implements IExtensionMessageSender {
 
     public sendMessage(message: ExtensionMessage, args?: any[]): Q.Promise<any> {
         let deferred = Q.defer<any>();
-        let messageWithArguments: MessageWithArguments = { message: message, args: args };
+        let messageWithArguments: MessageWithArguments = { message: message, args: args || [] };
```

Comment:
args is marked as mandatory in: MessageWithArguments.
Another option would be to mark it as optional, and make sure all the code expects it to possible be null.